scenes/src/app/scenes/slide-toggle/slide-toggle-scene.scss

```diff
@@ -0,0 +1,26 @@
+.wrapper {
```

Comment:
We should prefix all CSS selectors, either with `docs-` or with `scene-`

(in this and other PRs)